grails-app/domain/org/broadinstitute/orsp/IssueExtraProperty.groovy

```diff
@@ -40,6 +42,10 @@ class IssueExtraProperty {
     public static final String DATA_SHARING_BROAD = "dataSharingBroad"
     public static final String DATA_SHARING_COMMENTS = "dataSharingComments"
     public static final String DATA_SHARING_NIH = "dataSharingNih"
+    public static final String COLLABORATORS = "collaborators"
```

Comment:
With the addition of collaborators, `PROJECT_MANAGER` becomes unused. This might require a db migration to move all PMs to something else.